TaskLayer/MetaMorpheusTask.cs

```diff
@@ -226,9 +237,302 @@ protected internal void WritePsmsToTsv(List<NewPsmWithFdr> items, string outputF
             SucessfullyFinishedWritingFile(writtenFile, nestedIds);
         }
 
-        #endregion Protected Internal Methods
+        protected internal void WritePsmsToMzIdentmL(List<NewPsmWithFdr> items, List<DbForTask> dbFilenameList, string outputFolder, string fileName, List<string> nestedIds)
+        {
+            XmlSerializer _indexedSerializer = new XmlSerializer(typeof(mzIdentML.Generated.MzIdentMLType));
+            var _mzid = new mzIdentML.Generated.MzIdentMLType()
+            {
+                version = "1.2.0"
```

Comment:
Are you sure mzlib can write mzidentml version 1.2.0? I think it currently does 1.1.1